test/rekt/features/broker/source_to_sink.go

```diff
@@ -91,73 +95,29 @@ func SourceToSinkWithDLQ(brokerName, sinkName string, prober *eventshub.EventPro
 		panic(fmt.Errorf("can not find event files: %s", err))
 	}
 
-	f.Setup("install trigger", trigger.Install(via, brokerName, trigger.WithSubscriber(nil, "bad://uri")))
-
-	// Resources ready.
-	f.Setup("trigger goes ready", trigger.IsReady(via))
-
-	// Install events after data plane is ready.
-	f.Setup("install source", prober.SenderInstall("source"))
-
-	// After we have finished sending.
-	f.Requirement("sender is finished", prober.SenderDone("source"))
-
-	// Assert events ended up where we expected.
-	f.Stable("broker with DLQ").
-		Must("accepted all events", prober.AssertSentAll("source")).
-		Must("deliver event to DLQ", prober.AssertReceivedAll("source", sinkName))
-
-	return f
-}
+	f.Setup("install dlq", prober.ReceiverInstall(dlq))
 
-// SourceToTwoSinksWithDLQ tests to see if a Ready Broker acts as middleware.
-//
-// source ---> broker +--[trigger<via1>]--> bad uri
-//                |   |
-//                |   +--[trigger<vai2>]--> sink2
-//                |
-//                +--[DLQ]--> sink1
-//
-func SourceToTwoSinksWithDLQ(brokerName, sink1, sink2 string, prober *eventshub.EventProber) *feature.Feature {
```

Comment:
Why are we dropping this test?

Reply:
Basically is not testing anything new, and I'm suspecting that the resources it needs makes the tests fail

Reply:
The topology is different from the other test.

Reply:
it is testing that sink2 gets all the events even though all sink1 events where sent to the dlq

```
Must("deliver event to sink (via2)", prober.AssertReceivedAll("source", sink2))
```

Reply:
which is different from the other one.

Reply:
Yup, but it does not add anything new that is not already tested on the trigger's test nor the broker's test separately.

Reply:
It's testing that one bad trigger doesn't impact a different trigger.

Reply:
please link a similar test.

Reply:
Maybe we should add a similar test to the Trigger's suite, cause there is where I think a test <<like>> this one belongs, if that's ok with you? But first, let's pass this one that fixes the flakiness in the Broker's suite

Reply:
> Checked the Trigger's suite and there is no similar test.

I don't see the reason why we're dropping this test

> Maybe we should add it to the Trigger's suite, cause there is where I think a test <> this one belongs, if that's ok with you?

No, I don't understand why we're dropping this test to then re-add it? 
Let's just keep it and then move wherever is better

Reply:
Cause this is the flaky one

Reply:
So, we aren't really fixing the original issue ...

Reply:
I was thinking on dividing/refactoring this test into the Triggers suite (cause is testing Trigger's stuff), should I do that in this PR?

Reply:
Pushed the test, let's see if it works now